### PR TITLE
Fix UI issues with the map view in the chat.

### PR DESCRIPTION
**Pull request by mayank408**
Fixes issue #651 
Alignment of text view is corrected.

Screenshots for the change: 

screenshot_1496493749

### CODE REVIEW

File: app/src/main/res/layout/item_susi_map.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
                 <TextView
                     android:id="@+id/text"
-                    android:layout_width="wrap_content"
+                    android:layout_width="240dp"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/item_susi_map.xml]: Hardcoding width is not the best idea. On tablets this will look really small

Comment (iamareebjamal) [inline app/src/main/res/layout/item_susi_map.xml]: The ultimate issue is with gravity, right?

Comment (amitiwary999) [inline app/src/main/res/layout/item_susi_map.xml]: I think issue is text s shown at right side instead of left side.

Comment (amitiwary999) [inline app/src/main/res/layout/item_susi_map.xml]: @iamareebjamal So yes issue is with gravity of text.

Comment (iamareebjamal) [inline app/src/main/res/layout/item_susi_map.xml]: Yeah, that's about gravity of the text. Changing it to constant is not the best way to solve this. Try setting `layout_gravity` on the container layout or `gravity` on the text itself

Comment (mayank408) [inline app/src/main/res/layout/item_susi_map.xml]: The Problem is we are using card view of fixed size ie 250dp

Comment (iamareebjamal) [inline app/src/main/res/layout/item_susi_map.xml]: Then `match_parent` should do it. The problem with hardcoding is that if we change one of the dimens, we will have to change all and that will be difficult to track

Comment (mayank408) [inline app/src/main/res/layout/item_susi_map.xml]: @iamareebjamal We are using custom layout for this, therefore `match_parent`is not working as excepted. Could you help me out in this.

Comment (iamareebjamal) [inline app/src/main/res/layout/item_susi_map.xml]: Sure, I'll pull your changes and have a look

**Pull request closed without merging**